go.mod

```diff
@@ -7,12 +7,14 @@ toolchain go1.22.6
 require (
 	github.com/evanphx/json-patch v5.9.0+incompatible
 	github.com/ghodss/yaml v1.0.0
+	github.com/labstack/echo/v4 v4.12.0
```

Comment:
This dependency doesn't seem used, is it?

Reply:
Yeah, you're right, was just testing some stuff with it.